src/pysdk/types.py

```diff
@@ -205,6 +205,8 @@ class TimeInForce(Enum):
 class Venue(Enum):
     # the trade is cleared on the orderbook venue
     ORDERBOOK = "ORDERBOOK"
+    # the trade is cleared on the RFQ venue
+    RFQ = "RFQ"
```

Comment:
Temp patch, since DB is returning this venue still. 
I'll have to remove this from ReferenceData.